add-tourpackage.php

```diff
@@ -130,22 +124,22 @@
                             <div class="col-12 col-md-8 col-lg-8">
                                 <div class="row">
                                     <div class="col-6 col-md-4 col-lg-4">
-                                        <input type="radio" name="duration"> 1 Day
+                                        <input type="radio" name="duration" value="1 day"> 1 Day
```

Comment:
### Don't hard code this input fields and value attributes in this fields, 
You have to add this data to a table in application database, then you can put a iteration loop and generate this input fields and value attributes. This is important because if you want to add or update some of this attributes, You have to done this manually.